README.md

```diff
@@ -86,15 +88,15 @@ compiler support.
      # offset, which may be written in years, months, weeks, or days.
   - uses: dtolnay/rust-toolchain@master
     with:
-      toolchain: stable 18 months ago
+      toolchain: 'stable 18 months ago'
```

Comment:
The quotes are not necessary here, but it's a good habit to use quotes in YAML. Or as someone on HN said it: "save yourself pain. Always quote" (https://news.ycombinator.com/item?id=34509789).